build_parameters.json

```diff
@@ -164,7 +164,8 @@
 							"DESTINATION":     "step/controller:${BRANCH}-${ENVIRONMENT}-java-11",
 							"DESTINATION_PRD": "step/controller:${VERSION}-java-11",
 							"CONFIG": "controller-11-dockerfile",
-							"IMAGE_BASENAME": "step-controller-${VERSION}-java-11",
+							"IMAGE_BASENAME": "step-controller-${BRANCH}-${ENVIRONMENT}-java-11",
+							"RAW_DIRECTORY": "/step/${SIMPLIFIED_BRANCH_NAME}/sbom/",
```

Comment:
This seems to be a directory dedicated to SBOM. I'd suggest to call it SBOM_DIRECTORY then... (Note: also applies to all other occurrences of RAW_DIRECTORY, obviously).